robomagellan:
  hardware_interface:
    loop_hz: 10
    joints:
      - left_wheel
      - right_wheel
